js/tests/README.md

```diff
@@ -0,0 +1,17 @@
+## How does Bootstraps's test suite work?
+
+Bootstrap uses [QUnit](http://api.qunitjs.com/), a powerful, easy-to-use JavaScript unit test suite. Each plugin has a file dedicated to it's tests in `js/unit/[plugin-name].js`.
+
+* `/test/unit/` contains the unit test files for each Bootstrap plugin.
+* `/test/vendor/` contains third-party testing-related code for the browser environment (jQuery and QUnit)
+* `/test/visual/` contains visual test code which is manually evaluated.
+
+To test usage in a Node.js environment, tests are executed using `npm test`.
```

Comment:
"using `grunt test-js`."
The `test` Grunt task runs a bunch of other stuff in addition to the JS unit tests.